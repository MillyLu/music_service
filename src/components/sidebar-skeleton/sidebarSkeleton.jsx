import * as Styled from './styles';

export function SidebarSkeleton() {
    return(
        <Styled.SidebarItem>
            <Styled.SidebarSkeleton></Styled.SidebarSkeleton>
        </Styled.SidebarItem>
    )
}
