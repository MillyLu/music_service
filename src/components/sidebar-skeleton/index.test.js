/* eslint-disable no-undef */
import { render } from "@testing-library/react"

import { SidebarSkeleton } from "./sidebarSkeleton"


describe('<SidebarSkeleton/>' , () => {
    it('should render and match the snapshot', () => {
        const {container} = render(<SidebarSkeleton/>);
        expect(container).toMatchSnapshot();
    })
})