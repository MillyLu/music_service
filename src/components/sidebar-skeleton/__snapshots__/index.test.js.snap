// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SidebarSkeleton/> should render and match the snapshot 1`] = `
<div>
  <div
    class="sc-bcXHqe jVpBGY"
  >
    <div
      class="sc-gswNZR eHtCbv"
    />
  </div>
</div>
`;
